go/cmd/pfdhcp/utils.go

```diff
@@ -219,6 +219,9 @@ func ShuffleGateway(ConfNet pfconfigdriver.RessourseNetworkConf) (r []byte) {
 		if ConfNet.Type == "inlinel2" && ConfNet.NatEnabled == "disabled" {
 			return []byte(net.ParseIP(ConfNet.Gateway).To4())
 		}
+		if ConfNet.Gatewayvip != "" {
```

Comment:
Being mr. picky here but GatewayVIP would be a better name for this.

Even better ForceGatewayVIP would make it even more explicit